ssz/README.md

```diff
@@ -1 +1,67 @@
 # Simple Serialization (SSZ)
+
+Simple Serialization is a serialization protocol described by Vitalik Buterin with design rationale outlined [here](https://github.com/ethereum/beacon_chain/tree/master/ssz). --update this link to reflect design rationale document when common document is ready--
+
+SSZ To-Do:  
+[X]Move tests to tests/ssz  
+[ ]Add full test coverage for supported types  
+[ ]Refactor ssz.py  
```

Comment:
link to issue for both of these to dos https://github.com/ethereum/beacon_chain/issues/100